### DISCUSSION TITLE
Install DIA-NN on Red Hat

### DISCUSSION THREAD

**Discussion by haianhle** [vdemichev/DiaNN#489] [General]
Hi Vladim,

Thank you so much for always responding so quickly to everyone's questions. I really appreciate it following the discussions and learn a lot.

I'm trying to install DIA-NN on an EC2 running on Red Hat. What's your advice on how to proceed? Thank you so much.

Best wishes,
Anh

**Comment by vdemichev** [comment]
Hi Anh,

Just install as usual on any other Linux? There are rpm and deb packages provided.

Best,
Vadim

**Reply by haianhle** [reply]
Hi Vadim, thanks so much for your response. I apologized for misspelling your name, and I'm new to this, so my apology for the question! I will try the deb package. 

Best wishes,
Anh

**Comment by onurserce** [comment]
Hi Vadim,

The newest release 1.8.1 doesn't have an rpm package. I am trying to install DIA-NN on a HPC system but have no luck so far.

Could you provide an rpm package or more specific instructions as to how to install it using the source code?

Best regards,
Onur

**Reply by vdemichev** [reply]
Hi Onur,

Please just unpack the .tar.gz archive in the folder of your choice. This will work on a sufficiently recent Linux.

Best,
Vadim

**Reply by onurserce** [reply]
Thanks for your quick reply.

It turns out that the glibc of the HPC was too old, I have switched to another HPC and it seems to be working.

**Reply by heejongkim** [reply]
Hi,

I'm having the same issue. It would be tremendously helpful if I can get some hints of compiling dependencies, such as gcc version,  glibc version, and if any other libraries are needed to properly operate the software.

Thank you.

**Reply by vdemichev** [reply]
Hi, 

The process got updated for 1.9.1, please see here: https://github.com/vdemichev/DiaNN?tab=readme-ov-file#installation

Best,
Vadim
